connectors/agent/connector_record_manager.py

```diff
@@ -56,6 +56,7 @@ async def ensure_connector_records_exist(self, agent_config, connector_name=None
                         connector_id=connector_id,
                         service_type=service_type,
                         connector_name=connector_name,
+                        is_native=True,
```

Comment:
So we've settled on `is_native` flag reusage in the end?

Reply:
This is what I understood from the "mega-thread" we've had last week, will DM you the link